### PR TITLE
Add guide for string substitution conditional

### PR DESCRIPTION
**Pull request by m-mujica**
branched off of https://github.com/stealjs/steal-conditional/pull/27

### CODE REVIEW

File: docs/pages/substitution-conditional-loading.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+Reload _index.html_ to see your changes.
+
+![screen shot 2017-01-05 at 16 14 46](https://cloud.githubusercontent.com/assets/724877/21694104/5a33f32e-d362-11e6-8468-cafd83b0a021.png)
+
+If you reload _index.html_ a couple more times, you should see the other translations as well:
+
+![screen shot 2017-01-05 at 16 14 39](https://cloud.githubusercontent.com/assets/724877/21694124/6e1d4b60-d362-11e6-9d67-32ac20cbc0f3.png)
+
+Pretty cool, don't you think?
```
Comment (matthewp) [inline docs/pages/substitution-conditional-loading.md:258]: You might make a note that the reason for showing a random language is for demonstration, and if you *really* want to implement localization, use `navigator.language`.

Comment (m-mujica) [reply docs/pages/substitution-conditional-loading.md:258]: screen shot 2017-01-06 at 09 49 20

👍  👎 ?

Comment (m-mujica) [reply docs/pages/substitution-conditional-loading.md:258]: links to https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language

Comment (matthewp) [reply docs/pages/substitution-conditional-loading.md:258]: Yep, I like that.

**Comment by matthewp** [comment]
👍 merge when ready.

**Pull request merged**